tests/should_succeed/returnvalue.jou

```diff
@@ -0,0 +1,9 @@
+declare printf(pattern: byte*, ...) -> int 
+
+def pi() -> float:
+    pi = 3.1415 # 3.1415926535897932384626433832079
+    return pi # Return value 

```

Comment:
This should fail, because `pi` is of type `double` but the function should return a `float`. You shouldn't lose precision implicitly.